travel-agency-api/src/main/java/cz/muni/fi/pa165/bluebat/facade/TripFacade.java

```diff
@@ -8,8 +8,28 @@
  **/
 
 public interface TripFacade {
```

Comment:
Možná bych ještě přidal getTripList(), který by vrátil výpis všech Tripů pro nějaké list view -> pro tyto účely by taky nebylo špatné vytvořit TripListDTO, které by obsahovalo jen nejnutnější parametry (třeba název, destinaci, datum od/do a samozřejmě id)